src/api/rule-engine/Rule.ts

```diff
@@ -299,10 +300,10 @@ export class RuleService extends CoreWebService {
     });
   }
 
-  loadRules():Observable<RuleModel[]|CwError> {
+  loadRules( site:Site ):Observable<RuleModel[]|CwError> {
```

Comment:
We don't send the site with every request, but We need send the siteId because the end point need it.